.github/scripts/run_smartnet.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+set -e
+
+export NODE_IMAGE=public.ecr.aws/p6e8q1z1/aleph-node:31b1428-smartnet
+# key derived from "//0"
+export NODE_ID=5D34dL5prEUaGNQtPPZ3yN5Y6BnkfXunKXXz6fo7ZJbLwRRH
+export ALICE=5GrwvaEF5zXb26Fz9rcQpDWS57CtERHpNehXCPcNoHGKutQY
+
+mkdir -p docker/data/
+
+# generate chainspec
+docker run --rm -v $(pwd)/docker/data:/data --entrypoint "/bin/sh" -e RUST_LOG=debug "${NODE_IMAGE}" -c \
+       "aleph-node bootstrap-chain --base-path /data --account-ids $NODE_ID --faucet-account-id $ALICE --sudo-account-id $NODE_ID --chain-id a0smnet --token-symbol SZERO --chain-name 'Aleph Zero Smartnet' > /data/chainspec.smartnet.json"
+
+# Generate bootnode peer id
+export BOOTNODE_PEER_ID=$(docker run --rm -v $(pwd)/docker/data:/data --entrypoint "/bin/sh" -e RUST_LOG=info "${NODE_IMAGE}" -c "aleph-node key inspect-node-key --file /data/$NODE_ID/p2p_secret")
+
+# echo $BOOTNODE_PEER_ID
+
+docker-compose -f docker/smartnet-compose.yml up --remove-orphans
+
+exit $?
```

Comment:
I will admit to kinda cargo cult this across scripts - probably not always necessary, but I'd say it's good to make sure that e.g. CI picks up on any failures and fails the pipeline here and there instead of just limping to the next step ;)